src/renderer/StartupTasks.ts

```diff
@@ -5,6 +5,9 @@
 import * as Render from "./Render";
 import { showErrorToast } from "./util";
 
+import { AuthModal } from "./components/AuthModal";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['AuthModal' is defined but never used.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5164335)